src/custom/SetupPrerequisite/style.tsx

```diff
@@ -22,10 +22,10 @@ const ContainerCardWrapper = styled('div')(({ theme }) => ({
 }));
 
 const Card = styled('a')(({ theme }) => ({
+  flex: '1',
   padding: '2rem',
   textDecoration: 'none',
   background: theme.palette.mode === 'light' ? '#EEEEEE' : '#212121',
```

Comment:
why this is hard coded colors🤔?

Reply:
I think Akshay did it because the palette colors were lighter than he required.
![image](https://github.com/user-attachments/assets/5ef5bd99-0d64-4638-99d6-b414ef714e0a)